### PR TITLE
#16453 log error if env does not exists, throw exception if there are…

### PR DESCRIPTION
**Pull request by erickgonzalez**
… no env, remove annotation so does not show the PP dialog

### CODE REVIEW

**Review by nollymar — APPROVED** [review]

**Review by dsilvam — APPROVED** [review]

File: dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/PushNowActionlet.java [hunk 1]
```diff
@@ -73,15 +72,21 @@ public void executeAction(WorkflowProcessor processor, Map<String, WorkflowActio
...
                         envsToSendTo.add(e);
+                    }else{
+                        Logger.error(PushNowActionlet.class, "The Environment " + name + " does not exists");
                     }
                 }
 
             }
 
+            if(envsToSendTo.isEmpty()){
+                throw new DotPublisherException("There are no enviroments to send the bundle");
```
Comment (jcastro-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/PushNowActionlet.java]: Can you add the Contentlet's Identifier here please? It'd really help for support. Maybe update the error message to something like this:
```
"There are no enviroments to send the bundle containing Contentlet " + ref.getIdentifier()
```

File: dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/PushNowActionlet.java [hunk 2]
```diff
@@ -20,7 +20,6 @@
 import java.util.List;
 import java.util.Map;
 
-@Actionlet(pushPublish = true)
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/PushNowActionlet.java]: why did you remove this?

File: dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/PushNowActionlet.java [hunk 3]
```diff
@@ -73,15 +72,21 @@ public void executeAction(WorkflowProcessor processor, Map<String, WorkflowActio
             for (String name : whereToSend) {
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/PushNowActionlet.java]: set to final

File: dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/PushNowActionlet.java [hunk 4]
```diff
@@ -73,15 +72,21 @@ public void executeAction(WorkflowProcessor processor, Map<String, WorkflowActio
...
                 if (UtilMethods.isSet(name)) {
                     name = name.trim();
-                    Environment e = APILocator.getEnvironmentAPI().findEnvironmentByName(name);
+                    final Environment e = APILocator.getEnvironmentAPI().findEnvironmentByName(name);
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/PushNowActionlet.java]: rename to environment

File: dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/PushNowActionlet.java (hunk 1 above)
Comment (jdotcms) [reply dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/PushNowActionlet.java]: agree
